src/ripple/shamap/impl/SHAMapInnerNode.cpp

```diff
@@ -49,6 +49,10 @@ SHAMapInnerNode::SHAMapInnerNode(
 {
 }
 
+SHAMapInnerNode::~SHAMapInnerNode()
+{
+}
+
```

Comment:
Consider:  `SHAMapInnerNode::~SHAMapInnerNode() = default;`
(yes it actually works fine in a source)
Regardless, approved.

Reply:
I like that better, assuming it doesn't cause any problems.